### PR TITLE
header was added

### PR DESCRIPTION
**Pull request by ihor-soloviov**
[DEMO LINK] (https://ihor-soloviov.github.io/layout_miami/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,13 +1,339 @@
...
+                  required
+                >
+                <input
+                  name="email"
+                  type="email"
+                  class="contact-us__field form-field"
+                  placeholder="Email"
+                  required
+                >
+                <textarea name="message" class="contact-us__field form-field form-field--message"></textarea>
```
Comment (Avichez) [inline src/index.html:297]: Here is missing placeholder 'message' and the inputs using Arial font-family.
image

File: src/index.html [hunk 2]
```diff
@@ -1,13 +1,339 @@
...
+
+                <button class="button">Send</button>
```
Comment (Avichez) [inline src/index.html:299]: Needs to add `border: none;`

**Review by Avichez — CHANGES_REQUESTED** [review]
That's a great job 🔥 , especially animations on the first entry to the page.
1) Checklist 3, 6, in the deploy version I'm getting 404 error change in all Logos link to '#'.
2) For the menu needs to add `z-index` otherwise we will get this.
image
3) I'm not sure that you can change the colors of contacts, I understand that this is for animation, maybe to swap them?

File: src/index.html (hunk 1 above)
Comment (ihor-soloviov) [reply src/index.html:297]: Yap. here I had strange but. I don`t know why but my placeholder was invisible, so I just cutted this part. I'll change it and pray for its vision

**Review by alexandra-protyanova — CHANGES_REQUESTED** [review]
1. Checklist point 6
2. Let us follow the mockup, it is better to change color on hover)
Снимок экрана 2022-12-29 в 08 54 05

**Comment by alexandra-protyanova** [comment]
In some resolutions, form & `Explore` button stretch too wide, it is not critical, but it is better to give them `max-width`

**Comment by ihor-soloviov** [comment]
> 1. Checklist point 6
> 
>     2. Let us follow the mockup, it is better to change color on hover)
> 
> 
> Снимок экрана 2022-12-29 в 08 54 05

**Review by ArturErmolenko — APPROVED** [review]
Well done 👍

**Pull request closed without merging**